### DISCUSSION TITLE
How to access property called "class" as its a keyword in Python ?

### DISCUSSION THREAD

**Discussion by janaka44** [nazrulworld/fhir.resources#121] [Q&A] [answered]
Encounter resource has a property called "class" (Encounter.classHistory.class). How do we access such properties when its a keyword in Python ?

**Comment by nazrulworld** [comment, marked as answer]
You will use the ``class_fhir``  accessor instead of the python keyword ``class``.

**Reply by janaka44** [reply]
Thanks. btw, is there an API reference documentation of this ? I've read only the examples at the readme in homepage. That would solve a lot of my issues using this API. Thanks !

**Reply by nazrulworld** [reply]
Unfortunately, there is no example/api in the documentation like this, but you are welcome to add an example of your case.

**Comment by janaka44** [comment]
I think you misunderstood my question. I was asking about a documentation
of the whole API, not an example of how to use this class property.

Because there are some methods that I see in classes, so having a
documentation and examples would be helpful.
